src/app/mycomments.js

```diff
@@ -71,10 +76,31 @@ class MyComments extends React.Component {
 
       this.loadComments()
     })
+    this.loadUserOwnerComments()
   }
 
+  loadUserOwnerComments() {
```

Comment:
But is the difference to loadComments? I don't see it. It is tbasically he same, isn't it?

mycomments should not be changed for this task.

Reply:
It is not the same, sorry. I see it now.

But mycomments should not be changed for this task, just the index page.